src/components/logo/logo.component.tsx

```diff
@@ -1,5 +1,5 @@
-import * as React from "react";
-import { useConfig } from "@openmrs/esm-framework";
+import * as React from 'react';
```

Comment:
I think this project uses `import React from 'react'`. It does not have functional consequences - this is just for consistency.

Reply:
Everything should be consistent now.